v22.1/follower-reads.md

```diff
@@ -1,197 +1,191 @@
 
-To set `AS OF SYSTEM TIME follower_read_timestamp()` on all implicit and explicit read-only transactions by default, use one of the following options:
+With follower reads, any replica in a range can serve a read for a key as long as the time at which the operation is performed (i.e., the [`AS OF SYSTEM TIME`](as-of-system-time.html) value) is less than or equal to the range's closed timestamp.
 
-<ul>
-  <li>Set the <code>default_transaction_use_follower_reads</code> <a href="set-vars.html">session variable</a> to <code>on</code>. When <code>default_transaction_use_follower_reads=on</code>, all read-only transactions use exact staleness follower reads.</li>
-  <li>Execute the <code>SET SESSION CHARACTERISTICS AS TRANSACTION AS OF SYSTEM TIME follower_read_timestamp()</code> <a href="set-vars.html#special-syntax-cases">SQL statement</a>. This has the same effect as setting the session variable as shown above.</li>
-</ul>
+When a gateway node in a cluster receives a request to read a key with a sufficiently old [`AS OF SYSTEM TIME`](as-of-system-time.html) value, it forwards the request to the closest node that contains a replica of the data&mdash;whether it be a follower or the leaseholder.
 
- You can set `default_transaction_use_follower_reads` on a per-role basis; for instructions, see [Set default session variable values for a role](alter-role.html#set-default-session-variable-values-for-a-role).
-{{site.data.alerts.end}}
+For further details, see [An Epic Read on Follower Reads](https://www.cockroachlabs.com/blog/follower-reads-stale-data/).
 
-## Limitations
+### Limitations
 
 - [Exact staleness reads and long-running writes](#exact-staleness-reads-and-long-running-writes)
 - [Exact staleness read timestamps must be far enough in the past](#exact-staleness-read-timestamps-must-be-far-enough-in-the-past)
 - [Bounded staleness read limitations](#bounded-staleness-read-limitations)
 
-### Exact staleness reads and long-running writes
+#### Exact staleness reads and long-running writes
 
-Long-running write transactions will create write intents with a timestamp near when the transaction began. When an exact staleness follower read encounters a write intent, it will often end up in a "transaction wait queue", waiting for the operation to complete; however, this runs counter to the benefit exact staleness reads provide.
+Long-running write transactions will create [write intents](architecture/transaction-layer.html#write-intents) with a timestamp near when the transaction began. When an exact staleness follower read encounters a write intent, it will often end up in a "transaction wait queue", waiting for the operation to complete; however, this runs counter to the benefit exact staleness reads provide.
```

Comment:
I think "transaction wait queue" could be a link to https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue